docs/flink-connector.md

```diff
@@ -94,7 +94,24 @@ CREATE TABLE flink_table (
 );
 ```
 
-## Table managed in custom catalog
+## Table managed in Rest catalog
+
+The following SQL will create a Flink table in current Flink catalog, which maps to the iceberg table `default_database.flink_table` managed in Rest catalog.
+
+```sql
+CREATE TABLE flink_table (
+    id   BIGINT,
+    data STRING
+) WITH (
+    'connector'='iceberg',
+    'catalog-name'='rest_prod',
+    'catalog-type'='rest',
+    'uri'='https://localhost/'
+    -- credential, token is Optional
```

Comment:
I suggest linking to our FLink-ddl page so that users can see more optional configuration items. The same applies to Hive and Hadoop Catalog above.

Reply:
@hililiwei Should I delete flink-connector.md? It's explained on the FLink-ddl page, so flink-connector.md seems unnecessary.

Reply:
Ok

Reply:
I think this md is duplicate with 'flink-getting-started.md' and 'flink-ddl.md'. It looks unnecessary. #7246